liblastlog2/man/ll2_update_login_time.3.adoc

```diff
@@ -0,0 +1,62 @@
+//po4a: entry man manual
+= ll2_update_login_time(3)
+:doctype: manpage
+:man manual: Programmer's Manual
+:man source: util-linux {release-version}
+:page-layout: base
+:lib: liblastlog2
+:firstversion: 2.39
+
+== NAME
+
+ll2_update_login_time - Writes an *new* entry with updated login time.
+
+== SYNOPSIS
+
+*#include <lastlog2.h>*
+
+*int ll2_update_login_time (const char *__lastlog2_path__,
+			    const char *__user__, int64_t __ll_time__,
+			    char **__error__);*
+
+== DESCRIPTION
+
+Writes an *new* entry to database _lastlog2_path_ for user _user_.
+Time is set by _ll_time_ whereas the other values are taken from
+an already existing entry.
+
+--------------------------------------
+char    *error = NULL;
+const   char *db_path = "/var/lib/lastlog/lastlog2.db";
+const   char *user = "root";
+int64_t login_time = time(0); // Get the system time;
+
+int ret = ll2_update_login_time (db_path, user, login_time, &error);
+--------------------------------------
+
+== RETURN VALUE
+
+Returns 0 on success, -1 on failure. _error_ contains an error string if the return value is -1.
```

Comment:
It can also be `NULL` if the return value is `-1`.

Reply:
fixed